sdk/storage/Azure.Storage.Files/swagger/file-2019-02-02.json

```diff
@@ -543,7 +543,7 @@
                                 "description": "A UTC date/time value generated by the service that indicates the time at which the response was initiated."
                             },       
                             "x-ms-file-permission-key": {
-                                "x-ms-client-name": "FilePermissonKey",
+                                "x-ms-client-name": "FilePermissionKey",
```

Comment:
I also have a PR to update the Files Swagger in XStore

Reply:
Please revert this and add a transform to the README.md.  We're not making any changes to the swagger files now.  It was a lot of work to get them all to the same version and we have to keep them that way.

Reply:
The same for the rest of the changes in this file.

Reply:
The file swagger we have does not function correctly.  
- SetDirectoryProperties does not function correctly, as it is missing the restype=directory query parameter
- GetFile 206 response status is missing the file attributes, file creation time, file last write time, file permission key, file id, and parent id headers.  GetFile generally returns a 206.

We need to make these change for the the SDKs in all languages to function correctly.

/cc @amishra-dev

Reply:
Sure, I understand, but as part of the migration to azure-rest-api-specs it complicates things if .NET switches back to a different version.  I don't want to block that work on both this PR and the XStore PR going through.

Reply:
The XStore PR is going through today, and this PR could go through today, too.

Reply:
The XStore PR is merged.